src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,26 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+
+// Types
+import { Comment } from '../../Types/comment';
+
+// Components
+import { CommentInfo } from '../CommentInfo';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map((comment: Comment) => (
+      <CommentInfo
```

Comment:
Use destructuring assignment when mapping over the comments array to improve readability